moo/src/app.js

```diff
@@ -0,0 +1,36 @@
+import express from "express";
+import dotenv from "dotenv";
+import productRouter from "./routers/productRouter.js" 
+import articleRouter from "./routers/articleRouter.js"
+import commentRouter from "./routers/commentRouter.js"
+import cors from "cors"
+import uploadRouter from "./routers/uploadRouter.js";
+
+dotenv.config();
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json()); 
+app.use(cors({
+    origin: "*"
+}));
+
+app.use("/files", express.static("uploads"));
+
+app.get('/', (req,res)=> {
+    res.send("미션 3 홈페이지입니다");
+});
+
+app.use('/products', productRouter); 
+app.use('/articles', articleRouter);
+app.use('/comments', commentRouter);
+app.use('/files', uploadRouter);
+
+app.all(/(.*)/, (req, res) => {
+    res.status(404).send({ message: '요청하신 리소스를 찾을 수 없습니다.' });
+});
+
+app.listen(3000, ()=> {
+    console.log("서버를 시작하겠습니다-");
```

Comment:
나중에 개발하다보면 여러 포트에서 프로세스를 실행할 때도 있어서 포트까지 포함시켜주세요
```suggestion
    console.log(`서버가 포트 ${port}에서 실행 중입니다`);
```